### ISSUE TITLE
Removing a normalScrollElement causes to break "scroll to next section".

### ISSUE THREAD

**Issue by claverwebdesign** [alvarotrigo/fullPage.js#3450]
### Description
Removing a normalScrollElement causes to break "scroll to next section".
-- Disabling ScrollOverflow does not fix this.
-- only happening when normalScrollElements are removed from the DOM.
-- Scrolling inside iScroll elements keeps working.
-- Click to navigate to section keeps working

### Steps to reproduce it
1. Load overlay for Age Validation
2. fullPage.js --> normalScrollElements target this "overlay div" to enable scroll.
2. Validate Age and proceed to website
3. Javascript will remove the "overlay div" after validation.
4. Scrolling / swiping to next page breaks. 

### Versions 
Fullpage.js 3.0.4 (dev)

**Comment by alvarotrigo** [comment]
Thanks for reporting it!
I'll check it whenever I can!

**Comment by alvarotrigo** [comment]
Could you please provide  an isolated reproduction in [jsfiddle](https://jsfiddle.net/alvarotrigo/ea17skjr/) or [Codepen](https://codepen.io/alvarotrigo/pen/qqabrp), with no CSS or JS files external to fullPage.js and the minimum amount of HTML code. Use empty sections unless strictly necessary for the reproduction.

Perhaps you might find useful to read my article regarding [how to create isolated reproductions](https://alvarotrigo.com/blog/how-not-suck-at-asking-coding-questions/).

**Comment by claverwebdesign** [comment]
Hi @alvarotrigo,

I've managed to reproduce the issue: https://codepen.io/claverwebdesign/pen/MPOXgW 

Please have a look. 
Try to change $('#age-gate').remove(); to $('#age-gate').hide(); 
You will see that it doesn't break anymore.

**Comment by alvarotrigo** [comment]
@claverwebdesign steps to reproduce the issue?
After I click on "hide me" it seems to be working fine?

**Comment by DrQuarzZz** [comment]
I have encountered the same problem!

**My situation:**
I have overlay boxes (as normalScrollElements), that i clone and append outside the fullpage-wrapper. When i remove them again (for example with remove() but also when I just empty() them), I'm stuck on the current section when trying to scroll. The arrow Commands still work, but the event on the fullpage wrapper is gone.

**How to reproduce the issue in my Pen:**
Simply click the Button on a page and then close the overlay again with the red cross. You will now not be able to scroll to another section.

Here is a isolated pen that should show it all: [https://codepen.io/DrQuarzZz/pen/eQvoxw](https://codepen.io/DrQuarzZz/pen/eQvoxw)

**Things that are probably worth to mention:**
- When i take away the normalScrollElement in the fullpage initialization, everything works (except scrolling the overflow obviously)
- When i don't manipulate the DOM but instead translate the overlay outside the screen, the event returns (when moving the mouse) but only for desktop/mouse move. On Mobile, i'm still stuck in the current section
- This is kind of weird but when i translated the overlay just a little bit (5px-50px or sth), i could still change the sections when clicking on the background and not the overlay. But when i used bigger distances and the overlay started leaving the screen, it stopped to work everywhere.

**Where I encountered the problem:**
Desktop: Chrome, Firefox, Safari (macOS Sierra)
Mobile: Chrome, Firefox (LG Q6, Android 7.1.1)
It didn't occur on Edge Mobile, but i think that is due to the fact that normalScrollElement isn't working on there in general (or at least not for me).

Thanks for helping in advance ;)
